packages/ptokens-constants/src/index.ts

```diff
@@ -13,46 +9,8 @@ export enum BlockchainType {
   ALGORAND,
 }
 
-export const networkIdToTypeMap = new Map<string, BlockchainType>([
-  [NetworkId.GnosisMainnet, BlockchainType.EVM],
-  [NetworkId.BscMainnet, BlockchainType.EVM],
-  [NetworkId.PolygonMainnet, BlockchainType.EVM],
-  [NetworkId.GoerliTestnet, BlockchainType.EVM],
-])
-
-export enum Blockchain {
-  Ethereum,
-  Sepolia,
-  Goerli,
-  Bitcoin,
-  Eos,
-  Telos,
-  Bsc,
-  Xdai,
-  Polygon,
-  Ultra,
-  Fio,
-  Arbitrum,
-  Luxochain,
-  Fantom,
-  Algorand,
-  Libre,
-  Litecoin,
-  Gnosis,
-}
+export const networkIdToTypeMap = new Map<string, BlockchainType>([[ChainId.SepoliaTestnet, BlockchainType.EVM]])
```

Comment:
```suggestion
export const chainIdToTypeMap = new Map<string, BlockchainType>([[ChainId.SepoliaTestnet, BlockchainType.EVM]])
```

Reply:
Not sure about this mapping since it would force a certain BlockchainType given the chainId.
I would remove this and let the dev set the protocolId (here `BlockchainType`) at the `pTokensAssetBuilder` level instead for maximum flexibility.

This has also the advantage we don't need to maintain a mapping

Reply:
For now that only EVM it supported is fine. In the future there could be collisions between ChainId between evm and other types of blockchains.

Reply:
yeah but that's a check we dont' need here in my opinion, looks like more a last mile check (ie. dapp)